res/sectors.xsd

```diff
@@ -104,8 +104,11 @@
                       <xs:extension base="xs:string">
                         <xs:attribute name="Allegiance" type="xs:string" use="optional" />
                         <xs:attribute name="Hex" type="xs:unsignedShort" use="required" />
-                        <xs:attribute name="Color" type="xs:string" use="optional" />
+                        <xs:attribute name="Color" type="xs:string" use="required" />
```

Comment:
This change is based on the fact metadata.html doesn't have Color as optional. I'm not sure if that's an issue with the docs or the xsd.